docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,181 @@
+# ServiceWaterHeating - Rule 11-10
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** The service water heating system type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each ServiceWaterHeatingEquipment  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW Equipment is applicable
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_equip_dict = get_SWH_components_associated_with_each_SWH_bat(B_RMD)`
+- look at each SHW equipment in the building: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - CONTINUE TO RULE LOGIC
+- there are no cases where a Service Water Heating Equipment is not applicable to the rule
+- if there are no SWH equipment in the RMD, then this rule will not be evaluated
```

Comment:
if there is no SWH equipment then the outcome of this rule would be NOT_APPLICABLE